src/test/java/uk/gov/hmcts/reform/sscs/service/EvidenceShareServiceTest.java

```diff
@@ -9,14 +9,16 @@
 import java.net.URI;
```

Comment:
We should probably add tests to cover the 'non-pip' scenarios (as per AC) and the 'non-bulk-scan' scenarios (as per AC)